core/server/src/test/java/alluxio/worker/block/BlockWorkerTest.java

```diff
@@ -138,25 +127,6 @@ public void accessBlockTest() throws Exception {
   }
 
   /**
-   * Tests the Block Worker's session cleanable implementation.
-   */
-  @Test
-  public void cleanupSessionsTest() throws Exception {
```

Comment:
This test is removed because it isn't representative of how session cleaning is handled by the BlockWorker, and is tightly coupled to the implementation. If we want to test this we should inject a fake clock and make the session cleaner schedulable.